ParProcCo/msmapper_aggregator.py

```diff
@@ -49,7 +61,8 @@ def _write_aggregation_file(self, aggregation_output_dir: Path, output_data_file
             process = f.create_group("processed/process")
             process.attrs["NX_class"] = "NXprocess"
             f.create_dataset("processed/process/date", data=str(datetime.now(timezone.utc)))
-            f.create_dataset("processed/process/parameters", data=f"inputs: {output_data_files}, output: {aggregated_data_file}")
+            f.create_dataset("processed/process/parameters",
+                             data=f"inputs: {output_data_files}, output: {aggregated_data_file}")
             f.create_dataset("processed/process/program", data="ParProcCo")
             f.create_dataset("processed/process/version", data="1.0")
```

Comment:
Make this use a version string from the code base (which setup should use too): see for example https://github.com/DiamondLightSource/txrm2tiff/blob/4414454a459493c262644472b1dc41d91395101a/setup.py#L5